drivers/soundwire/intel.c

```diff
@@ -700,6 +700,25 @@ static int intel_params_stream(struct sdw_intel *sdw,
  * DAI routines
  */
 
+static int intel_free_stream(struct sdw_intel *sdw,
```

Comment:
Commit message:
Add the intel_free_stream() callback to deal with the change in IPC that
requires additional steps <del>need</del > to be done to clear the gateway node_id.